CHANGELOG.md

```diff
@@ -51,6 +51,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#833](https://github.com/nix-rust/nix/pull/833))
 - `ioctl_write_int!` now properly supports passing a `c_ulong` as the parameter on Linux non-musl targets
   ([#833](https://github.com/nix-rust/nix/pull/833))
+- Fixed `ptrace::Request` cfg conditions
```

Comment:
How about "Enabled more `ptrace::Request` definitions for uncommon Linux platforms"?

Reply:
I'll change to it.
Should it be moved to `Added` or `Changed` section?

Reply:
I moved it to `Changed` setcion.